demos/imagenet_classification_demo/main.cpp

```diff
@@ -0,0 +1,416 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+* @brief The entry point the Inference Engine sample application
+* @file imagenet_classification_demo/main.cpp
+* @example imagenet_classification_demo/main.cpp
+*/
+#include <vector>
+#include <queue>
+#include <memory>
+#include <string>
+#include <map>
+#include <condition_variable>
+#include <mutex>
+
+#include <inference_engine.hpp>
+
+#include <format_reader_ptr.h>
+
+#include <samples/common.hpp>
+#include <samples/slog.hpp>
+#include <samples/args_helper.hpp>
+#include <samples/classification_results.h>
+#include <samples/ocv_common.hpp>
+#include <samples/common.hpp>
+
+#include <ext_list.hpp>
+#include <vpu/vpu_plugin_config.hpp>
+#include <cldnn/cldnn_config.hpp>
+
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/highgui.hpp>
+
+#include <sys/stat.h>
+#include <ext_list.hpp>
+
+#include "imagenet_classification_demo.h"
+#include "grid_mat.hpp"
+#include "my_functor.hpp"
```

Comment:
Use `h` or `hpp` as suffix for all your headers.

Reply:
Done